spec/param_parser_spec.cr

```diff
@@ -2,101 +2,98 @@ require "./spec_helper"
 
 describe "ParamParser" do
   it "parses query params" do
-    route = Route.new "POST", "/" do |env|
-      hasan = env.params.query["hasan"]
-      "Hello #{hasan}"
-    end
     request = HTTP::Request.new("POST", "/?hasan=cemal")
     query_params = Kemal::ParamParser.new(request).query
     query_params["hasan"].should eq "cemal"
   end
 
-  it "parses multiple values for query params" do
-    route = Route.new "POST", "/" do |env|
-      hasan = env.params.query["hasan"]
-      "Hello #{hasan}"
+  context "#query" do
+    it "parses multiple values for query params" do
+      request = HTTP::Request.new("POST", "/?hasan=cemal&hasan=lamec")
```

Comment:
hmm.. What are your thoughts on this? 
IMO I would assume `params["hasan"]` would return `lamec`, but not an array of both. Sort of like doing
```
a = 1
a = 2
--VS--
a[] = 1
a[] = 2 
```
I'm not sure what the rules for this actually are though. I think since Ruby based apps turn query params in to a hash, it would be like `{"hasan" => "cemal", "hasan" => "lamec"}` which wouldn't work.